from_dict/_from_dict.py

```diff
@@ -25,6 +26,18 @@ def __repr__(self):
         return f"FromDictTypeError({self.location!r}, {self.expected_type!r}, {self.found_type!r})"
 
 
+class FromDictUnknownArgsError(ValueError):
+    def __init__(self, unknown_args: Dict[str, Any]):
```

Comment:
It's probably enough to just save the names of the fields, not their values. Contents can possibly be very large and/or not good to print and I'm not sure if it adda much information.